src/api/CommunicationsApiImpl.ts

```diff
@@ -137,10 +133,9 @@ class CommunicationsApiImpl implements CommunicationsApi {
                 }
             });
         } catch (e) {
-            call.write({
-                subscribeError: {
-                    reason: `Address ${call.request.topic.address} not found`
-                }
+            call.emit('error', {
```

Comment:
How this perform on the client-side. Is the stream is closed after that?

Reply:
On the client side, at least in Python, it raises an exception

Reply:
Now I understand why you ask that. I don't know, Ithink it doesn't, so I'll add a "call.end()" In order to prevent leaving the stream open